src/components/ControlBar/ControlBar.tsx

```diff
@@ -17,11 +18,17 @@ function ControlBar(props: ControlBarProps) {
     ctaButtonText,
     nextButtonHandler,
     isNextButtonDisabled,
+    instructions,
   } = props;
 
   return (
     <div className='cio-question-buttons-container'>
       {showBackButton && <BackButton onClick={backButtonHandler} />}
+      {instructions && instructions !== '' ? (
```

Comment:
isn't just `instructions` enough? since an empty string is false

Reply:
I think we can just do `{instructions && <div className='cio-question-buttons-container-text'>{instructions}</div>}`